### PR TITLE
Fixed version of #1

### PR DESCRIPTION
**Pull request by sergeybykov**
In https://github.com/dotnet/orleans-templates/commit/7bb88203ea9c83d9bcd4b29ca71a095732cfb85e I fixed project name and path in the .sln file to make #1 compile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,164 @@
-# orleans-templates
-Templates for build Microsoft Orleans based solutions
+[![NuGet](https://img.shields.io/nuget/v/Microsoft.Orleans.Templates.svg?style=flat)](http://www.nuget.org/profiles/Orleans)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dotnet/orleans?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+# Overview
+
+This project contains the Project to make it easier to getting started with [Orleans](https://github.com/dotnet/orleans).
```
Comment (ReubenBond) [inline README.md]: `This projects contains templates for getting started with [Orleans](https://github.com/dotnet/orleans).`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,164 @@
...
+
+The templates are leveraging the new [dotnet templating engine](https://github.com/dotnet/templating).
```
Comment (ReubenBond) [inline README.md]: no need to say 'new' here

File: README.md [hunk 3]
```diff
@@ -1,2 +1,164 @@
...
+
+# Provided templates
+
```
Comment (ReubenBond) [inline README.md]: We could drop this heading or copy the list of templates from below

**Comment by sergeybykov** [comment]
Addressed feedback.

**Comment by galvesribeiro** [comment]
What now? Nuget?

**Comment by sergeybykov** [comment]
I think so.

**Comment by sergeybykov** [comment]
Need to set up an automated build for it in VSTS.

**Pull request merged**